index.js

```diff
@@ -2,38 +2,28 @@ var exec = require('child_process').exec;
 
 module.exports = pushDir;
 
-function pushDir(opts) {
+function pushDir(args) {
   getLastCommitInfo().then(function(info) {
     var hash = info.hash;
     var detachedHead = info.branch === '';
     var originalBranch = detachedHead ? hash : info.branch;
 
-    var directory = opts.dir;
-    var local = opts.branch + '-' + hash;
-    var remote = opts.remote || 'origin';
-    var remoteBranch = opts.branch;
-    var message = typeof opts.message === 'function' ?
-      opts.message(hash) : opts.message || hash;
-    var allowUnclean = opts['allow-unclean'] || opts.force;
-    var overwriteLocal = opts['overwrite-local'] || opts.force;
-    var cleanup = opts.cleanup === undefined ? false : opts.cleanup;
+    var remoteSpecified = args._.length > 1;
+    var dirBranch = remoteSpecified ? args._[1] : args._[0];
+    var directory = dirBranch.split(':').slice(0, -1).join(':');
+    var remoteBranch = dirBranch.split(':').slice(-1)[0];
```

Comment:
I think this stuff actually probably belongs in `bin/push-dir.js`. For programmatic usage it seems silly to require the user pass in a colon-separated string rather than the explicit.

Plus, we probably want to log out usage if usage is incorrect, which is more convenient from `bin/push-dir.js`.